docs/content/latest/yugabyte-platform/manage-deployments/high-availability.md

```diff
@@ -0,0 +1,183 @@
+
+When you promote a standby platform to active, Yugabyte Platform restores your selected backup, and automatically demotes the previous active platform to standby mode.
+
+## Setting up an HA Cluster
+
+### Prerequisites
+
+* Yugabyte Platform v2.5.3.1 or above.
+* You have already [installed](../../install-yugabyte-platform/) multiple YB Platform instances to be used in the HA cluster.
+* Platform VMs are all able to connect to each other over the port that the Platform UI/API is normally reachable over (80/443, for example).
+* All platforms are running the same version of Yugabyte Platform software (it is good practice to upgrade all platforms in the HA cluster at close to the same time).
+
+### Set up the Active Platform
+
+1. Click the "Admin" tab at the bottom of the left-side navigation panel.
+
+    <br/>
+
+    ![Replication configuration tab](/images/yp/high-availability/replication-configuration.png)
+
+1. Select the `Active` instance type.
+
+    <br/>
+
+    ![Active instance type](/images/yp/high-availability/instance-type-active.png)
+
+1. Enter this platform’s IP address or hostname (including the HTTP or HTTPS protocol prefix).
+
+    <br/>
+
+    ![Add an instance](/images/yp/high-availability/add-active-instance.png)
+
+1. Generate a key.
+
+    <br/>
+
+    ![Generate a shared key](/images/yp/high-availability/generate-key.png)
+
+1. Select your desired replication frequency, in minutes.
+
+    <br/>
+
+    In most cases, you don't need to replicate very often. Yugabyte recommends... **DANIEL, WHAT IS THE REC HERE?**
+
+    <br/>
+
+    ![Replication frequency](/images/yp/high-availability/replication-frequency.png)
+
+1. Click "Create".
+
+1. Next, click the "Instance Configuration" tab at the top of the screen.
+
+    <br/>
+
+    The address for this platform should be the only entry row in the table.
+
+    <br/>
+
+    ![Instance list](/images/yp/high-availability/instance-configuration-active.png)
+
+1. Click "Add Instance" for each standby platform you would like to add to the HA cluster, and enter the standby platform’s IP address or hostname (including the HTTP/HTTPS protocol prefix).
+
+    <br/>
+
+    ![Add an instance](/images/yp/high-availability/add-standby-instance.png)
+
+### Set up Standby Platforms
+
+Once you've set up the active platform, you can set up one or more standby platforms.
+
+**Repeat the following steps for each standby platform** you wish to add to the HA cluster:
+
+1. Click the "Admin" tab at the bottom of the left-side navigation panel.
+
+    <br/>
+
+    You should be presented with a screen similar to the following:
+
+    <br/>
+
+    ![Replication configuration tab](/images/yp/high-availability/replication-configuration.png)
+
+1. Select the `Standby` instance type.
+
+    <br/>
+
+    ![Standby instance type](/images/yp/high-availability/instance-type-standby.png)
+
+1. Enter this platform’s IP address or hostname (including the HTTP or HTTPS protocol prefix).
```

Comment:
Or port (if applicable). Also the screenshot you have linked here is not correct, the modal is only for adding standby platforms on the active platform.